modules/core/ante/ante.go

```diff
@@ -29,25 +29,25 @@ func (ad AnteDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool, ne
 		for _, m := range tx.GetMsgs() {
 			switch msg := m.(type) {
 			case *channeltypes.MsgRecvPacket:
-				if _, found := ad.k.GetPacketReceipt(ctx, msg.Packet.GetDestPort(), msg.Packet.GetDestChannel(), msg.Packet.GetSequence()); found {
+				if response, err := ad.k.RecvPacket(ctx.Context(), msg); err == nil && response.Result == channeltypes.NOOP {
```

Comment:
I think you should be using `sdk.WrapSDKContext(ctx)` here in favour of `ctx.Context()`. 
I think this might be why you're seeing `interface conversion: interface {} is nil, not types.Context`

Reply:
Awesome! That fixed the problem. Thank you, @damiannolan!